gradle-mvn-push.gradle

```diff
@@ -4,6 +4,8 @@ apply plugin: 'signing'
 final isSnapshot = version.contains('SNAPSHOT')
 
 final skipTestFixtures() {
+	println "*** $project"
```

Comment:
Is this an intended change?

Reply:
Oops, nope. I'll remove this shortly. Thanks for being so attentive!